Scripts/build-phases/swiftlint.sh

```diff
@@ -18,14 +20,15 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 # Temporarily move to the root directory so that SwiftLint can correctly
 # find the paths returned from the `git` commands below.
-pushd $DIR/../../ > /dev/null
+pushd "$DIR/../../" > /dev/null || exit 1
 
-# Paths relative to the root directory
-SWIFTLINT="./vendor/swiftlint/bin/swiftlint"
-CONFIG_FILE=".swiftlint.yml"
+# Paths are now relative to the root directory
+SWIFTLINT_BIN="./Pods/SwiftLint/swiftlint"
+# Can't do $SWIFTLINT="$SWIFTLINT_BIN lint --quiet" to DRY because the value is then passed to xcargs
+SWIFTLINT_ARGS="lint --quiet"
```

Comment:
For consideration: the Pocket Casts folks preferred to add a `--lenient` option so that locally everything is a warning and not an error (while on CI, everything is an error with `--strict`), just so they can quickly experiment instead of being blocked by SwiftLint in addition to the compiler.

Reply:
I understand that rationale, although personally I'm in the opposite camp. I want to know everything that is wrong as soon as I can locally. I find it annoying when I have to go back to a PR to adjust the code because CI is stricter. 

Anyways, that's not really up for _me_ to decide, but for the Woo team. On that note, I'm pleased to see you opened p91TBi-aYg-p2